service/chartconfig/v1/helm/error.go

```diff
@@ -9,9 +17,16 @@ func IsInvalidConfig(err error) bool {
 	return microerror.Cause(err) == invalidConfigError
 }
 
-var notFoundError = microerror.New("not found")
+var releaseNotFoundError = microerror.New("release not found")
+
+// IsReleaseNotFound asserts releaseNotFoundError.
+func IsReleaseNotFound(err error) bool {
+	if strings.HasPrefix(err.Error(), releaseNotFoundErrorPrefix) {
```

Comment:
When working with errors within matchers it is always safer to only work with the cause like below. The error here can be wrapped and the message might not be what we expect because we do not match against the origin of the error. This is actually the task of this matcher though.